### PR TITLE
Changelogs

### PR DESCRIPTION
**Pull request by HaydarAk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,28 @@ All notable changes to this project will be documented in this file.
...
+
+### Added
+
+* `ids:ConfigurationManager` can refer to the configured App Resources, Brokers, App Stores and Clearing Houses represented as catalogs.
+* `ids:AppRoute` can refer to Broker and Clearing House instances (via `ids:appRouteClearingHouse` and `ids:appRouteBroker`), which are used for publishing and logging related communication.
+* `ids:GenericEndpoint` for non-IDS endpoints.
+* Catalog subclasses for Broker, AppStore, AppResource and ClearingHouse instances. Check [model/infrastructure/Catalog.ttl](Catalog.ttl) class for details.
+* Additional properties for Participants for identification and classification, e.g., legal name, business identifier, legal form and VAT ID (addressing requirements to participants in Gaia-X)
+* Additional properties for `ids:DataApp` and `ids:AppRepresentation` to express supported usage policies as well as app-related runtime, configuration and deployment information.
+* Extensive additions to the SHACL shapes.
```
[the remark this answers is not in the document]
Comment (HaydarAk) [reply CHANGELOG.md:19]: yes and no. Maybe I "merge" it with the 'Removal of idsm annotation properties' statement. 

The entry
`Extensive additions to the SHACL shapes.` 
is only half of the truth. The other half is the entry further below:
`Removed idsm metamodel annotations which were used to annotate cardinality for properties`

As you know, we now fully support SHACL in our java-lib generation process. Some of our colleagues did a great job and added a bunch of SHACL shapes (see Pull Request #401 ) which were missing.  We hopefully have (almost complete) 
 SHACL coverage now. At least we added shacl shapes for all properties, which had `idsm:` annotations before.

**Comment by maboeckmann** [comment]
Can you please add an existing property "defaultEndpoint" now became mandatory? This is a breaking change and should be mentioned.

**Comment by HaydarAk** [comment]
> Can you please add an existing property "defaultEndpoint" now became mandatory? This is a breaking change and should be mentioned.

You mean the property of the Connector class?
it was already mandatory in 4.0. 

See (starting from line 94): 
https://github.com/International-Data-Spaces-Association/InformationModel/blob/master/testing/infrastructure/ConnectorShape.ttl

**Comment by maboeckmann** [comment]
> > Can you please add an existing property "defaultEndpoint" now became mandatory? This is a breaking change and should be mentioned.
> 
> You mean the property of the Connector class?
> it was already mandatory in 4.0.
> 
> See (starting from line 94):
> https://github.com/International-Data-Spaces-Association/InformationModel/blob/master/testing/infrastructure/ConnectorShape.ttl

That is very interesting. This change only became visible in the broker a few weeks after the 4.0 release... Thanks for the clarification!

**Review by clange — APPROVED** [review]
Fine, thanks!

**Pull request merged**